htdocs/main.php

```diff
@@ -200,7 +200,7 @@ function tplFromRequest($param)
         . htmlspecialchars($e->getTraceAsString())
         . "</pre>";
 } catch(Exception $e) {
-    switch($e->getCode()) {
+    switch($e->getMessage()) {
```

Comment:
currently if you get timed out, you will get the message "you do not have access to this page" which might get confusing to users. the message should read something like
"You have been signed out due to inactivity. Please sign in to access this page"

Reply:
We could change the message but I don't know if 403.tpl was intended to be used only with public_layout.tpl . 
It is hard to determine if the client is anonymous because the session expired/logout or if he just never logged in. 
We could throw a 401 in the later case at `php/libraries/NDB_Caller.class.inc L#127` (This could be done in the next minor) 
@driusan ?

Reply:
Wasn't there some way that it was previously handled in javascript with a popup dialog rather than a different page?